pytorch_to_returnn/torch/nn/modules/linear.py

```diff
@@ -93,7 +93,11 @@ def __init__(self):
   def create_returnn_layer_dict(self, *inputs: Tensor, **kwargs) -> Dict[str, Any]:
     sources = [self._get_input_layer_name(source) for source in inputs]
     assert len(sources) == 2
-    return {"class": "dot", "from": sources}
+    red1 = inputs[0].returnn_naming_entry.get_returnn_axis_description(-1)
+    red2 = inputs[1].returnn_naming_entry.get_returnn_axis_description(-2)
+    var1 = inputs[0].returnn_naming_entry.get_returnn_axis_description(-2)
+    var2 = inputs[1].returnn_naming_entry.get_returnn_axis_description(-1)
```

Comment:
Better make this consistent as it is done in other layers. Via `self.get_axis...`

Reply:
I'm not sure what you mean here. I don't see modules which use a function `self.get_axis...`

Reply:
See how all other layers get the axis.

Reply:
`self._get_input_axis_to_returnn`